x-pack/plugins/spaces/public/experiments/is_solution_nav_enabled.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { from, of, shareReplay } from 'rxjs';
+
+export const SOLUTION_NAV_FEATURE_FLAG_NAME = 'solutionNavEnabled';
+import type { CloudExperimentsPluginStart } from '@kbn/cloud-experiments-plugin/common';
+import type { CloudStart } from '@kbn/cloud-plugin/public';
+
+export const isSolutionNavEnabled = (
+  cloud?: CloudStart,
+  cloudExperiments?: CloudExperimentsPluginStart
+) => {
+  return Boolean(cloud?.isCloudEnabled) && cloudExperiments
```

Comment:
In fact I just moved the setup that shared UX team added to a common file 🙂 But I can image that if we already run an experiment and change distribution for it we might later benefit from having observable.

Reply:
> In fact I just moved the setup that shared UX team added to a common file 🙂

Ah, I see!

> But I can image that if we already run an experiment and change distribution or have dynamic allocation for a target group we might later benefit from having observable.
Though it might be better to clarify how they intended to set this up.

Yeah, it feels a bit premature to optimize for something that might never happen. We can always migrate to Observable should `cloudExperiments` become more dynamic.